<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageBoxExX</class>
 <widget class="QDialog" name="MessageBoxExX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="TitleBar" name="titleBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="infoTE">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="iconL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QGroupBox</extends>
   <header location="global">src/utils/widget/titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
